ols/src/llms/llm_loader.py

```diff
@@ -16,6 +16,14 @@
 warnings.simplefilter("ignore", UserWarning)
 
 
+class MissingProvider(Exception):
+    """Exception thrown when provided provider is not specified."""
+
+
+class MissingModel(Exception):
+    """Exception thrown when model is not specified."""
```

Comment:
The Google style guide is promoting some syntax for exceptions docstrings - https://google.github.io/styleguide/pyguide.html#384-classes

So "Exception thrown when model is not specified." should be like "Model is not specified."

Reply:
Indeed, fixed